godel_scan_analysis/src/keyence_scan_server.cpp

```diff
@@ -0,0 +1,81 @@
+#include "godel_scan_analysis/keyence_scan_server.h"
+
+#include <pcl_ros/transforms.h>
+#include <pcl/filters/voxel_grid.h>
+
+// Constants
+const static double VOXEL_GRID_LEAF_SIZE = 0.005; // 5 mm
+
+const static double VOXEL_GRID_PUBLISH_PERIOD = 2.0; // seconds
+
+const static double TF_WAIT_TIMEOUT = 0.25; // seconds
+
+godel_scan_analysis::ScanServer::ScanServer(const std::string& world_frame, 
+                                            const std::string& scan_frame)
+  : map_(new ColorCloud)
+  , buffer_(new ColorCloud)
+  , from_frame_(world_frame)
+  , to_frame_(scan_frame)
+{
+  ros::NodeHandle nh;
+  scan_sub_ = nh.subscribe("profiles", 500, &ScanServer::scanCallback, this);
+  cloud_pub_ = nh.advertise<ColorCloud>("color_cloud", 1);
```

Comment:
@gavanderhoorn  all good points.  I'm not so sure that there are readability advantages as one could argue that in the godel way all topics can be easily found at the top of the source, if one knows what to look for.  As you mentioned though, in this project we should probably continue to use constants as most of godel has been written that way, but in other projects it'd be best to state topic strings inside the context where they are used as we always strive to be as ROS compliant as possible.